belgif-rest-problem-validator/src/test/java/io/github/belgif/rest/problem/validation/SsinValidatorTest.java

```diff
@@ -24,12 +24,20 @@ void okIncompleteDate() {
 
     @Test
     void okBis() {
+        // bis with month == 0 && day > 0
+        assertValidSsin("10000100182");
         // bis with month == 20
         assertValidSsin("92200006963");
+        // bis with month == 20 && day > 0 && day < 32
+        assertValidSsin("10200100128");
+        // bis with month == 20 && day > 31
+        assertValidSsin("24203200171");
```

Comment:
Not sure this can actually happen for BIS numbers? But I guess it's OK that it passes the checks.

Reply:
> Not sure this can actually happen for BIS numbers? But I guess it's OK that it passes the checks.

Verified it. It is accepted with cbss-commons